study-builder/studies/snippets/bool-question-yes-no.conf

```diff
@@ -15,6 +15,10 @@
           {
             "language": "en",
             "text": "Yes"
+          },
+          {
+            "language": "es",
+            "text": "Sí"
```

Comment:
Hmm since this is a shared snippet file, I think we should probably make a copy for MBC and add Spanish in that copy.

Reply:
Even though its shared other cmi studies use only english, right? Does this cause any issue ?